ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -1204,18 +1205,11 @@ class DamageTracker {
     this.ReloadTriggers();
   }
 
-  OnDataFilesRead(files) {
-    this.dataFiles = files;
-    this.ReloadTriggers();
-  }
-
   ProcessDataFiles() {
     // Only run this once.
     if (this.triggerSets)
       return;
-    // Wait until OnPlayerChange + OnDataFilesRead occur for the first time.
-    if (!this.dataFiles)
-      return;
+    // Wait until OnPlayerChange occurs for the first time.
```

Comment:
This comment was tied to [this commit](https://github.com/quisquous/cactbot/commit/c18076d36044720097ce23418a0f2749aa751e82#diff-d0c1e5eee090caf06bb314acbb06a90744cb98ad78f59ab62a542a67dd1beeaa), and can be deleted as well.
```suggestion
```